DancingGoat/Views/Shared/EditModePanel.cshtml

```diff
@@ -1,18 +1,31 @@
-﻿@model DancingGoat.Models.EditPanelViewModel
+﻿@using DancingGoat.Utils
+@model DancingGoat.Models.EditPanelViewModel
+@functions {
+    public string ProjectGuid => ProjectUtils.GetProjectId();
+}
 
 <div class="edit-mode-panel">
-    <div>
-        <div class="edit-mode-panel__toggle">
-            <label class="switch">
-                <input id="edit-mode-switch" type="checkbox" onchange="toggleEditMode(this.checked)" autocomplete="off">
-                <span class="slider round"></span>
-            </label>
+    <div class="edit-toggle">
+        <label class="edit-toggle__switch">
+            <input id="edit-mode-switch" type="checkbox" onchange="toggleEditMode(this.checked)" autocomplete="off">
+            <span class="slider round"></span>
+        </label>
+        <div class="edit-toggle__caption">
+            <span class="edit-toggle__title">Edit mode</span>
         </div>
-        <div class="edit-mode-panel__caption">
-            <span class="edit-mode-panel__title">Edit mode</span>
+    </div>
+    <div class="preview-toggle">
+        <label class="preview-toggle__switch">
+            <input class="preview-mode-switch" type="checkbox" onchange="togglePreviewMode(this.checked, '@ProjectGuid', true)
+                   " autocomplete="off">
+            <span class="slider round"></span>
+        </label>
+        <div class="preview-toggle__caption">
+            <span class="preview-toggle__title">Preview mode</span>
         </div>
     </div>
-    <div class="edit-mode-panel__navigate-link">
+
+    <div class="footer-navigate-link">
```

Comment:
Used to be Element, now it is incosistent, as it is footer in edit mode panel

Reply:
You might be right, renamed to:
`edit-mode-panel__navigate-link`